roles/ceph-infra/tasks/tune_kernel_parameters.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "tune kernel parameters {{ tune_kernel_parameters }} and merge with existing templates from /etc/grub.d"
+  command: >
+    grubby
+    --update-kernel=ALL
+    --copy-default
+    --args="{{ tune_kernel_parameters | quote }}"
+  changed_when: false
+
+- name: reboot the system to apply new kernel parameters
+  shell: sleep 2 && shutdown -r now
```

Comment:
You should switch the shell(shutdown)+wait_for+command(uptime) tasks by the reboot module [1]

[1] https://docs.ansible.com/ansible/latest/modules/reboot_module.html